### DISCUSSION TITLE
reveal uninstall token

### DISCUSSION THREAD

**Discussion by thomj78** [CrowdStrike/falconpy#1119] [Q&A] [answered]
Is there any plan to add endpoint for /policy/combined/reveal-uninstall-token/v1 ?

**Comment by jshcodes** [comment, marked as answer]
Hi @thomj78 -

You can find documentation for the [revealUninstallToken](https://falconpy.io/Service-Collections/Sensor-Update-Policy.html#revealuninstalltoken) operation here.

Let us know if there are any questions! 😄
